src/Uno.UI/UI/Xaml/Controls/ContentControl/ContentControl.cs

```diff
@@ -374,8 +374,6 @@ public void UpdateContentTemplateRoot()
 			{
 				OnApplyTemplate();
 			}
-
-			SyncDataContext();
```

Comment:
@MartinZikmund why was this particular statement problematic?

Reply:
It's duplicated. The only caller of `UpdateContentTemplateRoot` already does `SyncDataContext`.

Reply:
Yes, the call was duplicated, so the sync happened twice unnecessarily